src/Kentico.Xperience.Lucene.Core/Scaling/DeleteIndexWebFarmTask.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CMS.Base;
+using CMS.Core;
+
+using Kentico.Xperience.Lucene.Core.Indexing;
+
+namespace Kentico.Xperience.Lucene.Core.Scaling;
+internal class DeleteIndexWebFarmTask : WebFarmTaskBase
```

Comment:
Please add empty line between class and namespace. Also add XML doc explaining what for is this task.